@osschar , clang complains 
```
Utilities/XrdAdaptor/src/XrdRequestManager.cc:90:3: warning: explicitly defaulted default constructor is implicitly deleted [-Wdefaulted-function-deleted]
```
see the log https://cmssdt.cern.ch/SDT/jenkins-artifacts/pull-request-integration/PR-1dd22a/19266/clang-logs/Utilities/XrdAdaptor/log.html . Can you please fix this warning?

--- a/Utilities/XrdAdaptor/src/XrdRequestManager.cc
+++ b/Utilities/XrdAdaptor/src/XrdRequestManager.cc
@@ -87,7 +87,7 @@ class SendMonitoringInfoHandler : public XrdCl::ResponseHandler {
 public:
   SendMonitoringInfoHandler(const SendMonitoringInfoHandler &) = delete;
   SendMonitoringInfoHandler &operator=(const SendMonitoringInfoHandler &) = delete;
-  SendMonitoringInfoHandler() = default;
+  SendMonitoringInfoHandler() = delete;
 
   SendMonitoringInfoHandler(const std::string &url) : m_fs(url) {}
 

